tst/worksheets.tst

```diff
@@ -5,16 +5,9 @@ gap> START_TEST( "worksheets.tst" );
 
 #
 gap> AUTODOC_TestWorkSheet("general");
-Extracting manual examples for General Test package ...
-2 chapters detected
-Chapter 1 : extracted 2 examples
-Chapter 2 : no examples 
```

Comment:
Any particular reason to turn off the info messages in the test suite here? I actually found it helpful to see this for debugging purposes...

Reply:
No reason in particular!

Normally I turn them off in test suites because they contain things like paths or random elements that are difficult to reproduce. But in this case I don't think that's a problem.

I'll turn them back on, and also fix something else I just spotted…

Reply:
They're back on in the test suite, which actually revealed a mistake I made, which I've now fixed. So they've already been useful!